data/os/RedHat.yaml

```diff
@@ -2,4 +2,4 @@
 tang::package_name:
   - 'tang'
 
-tang::service_name: 'tang.socket'
+tang::service_name: 'tang.service'
```

Comment:
Did you check if this works?
It looks like the package only provide a multiinstance unit: https://centos.pkgs.org/8-stream/centos-appstream-x86_64/tang-7-8.el8.x86_64.rpm.html

Reply:
It should work: /usr/lib/systemd/system/tangd@.service